### PR TITLE
Adding  GINA, ethics timeline, cleaning up ethics ch

### PR DESCRIPTION
**Pull request by cansavvy**
#### What changes are being implemented in this Pull Request?

Still in progress. Trying to learn about GINA specifics

### CODE REVIEW

File: 05-Data_Ethics.Rmd [hunk 1]
```diff
@@ -159,39 +159,76 @@ Possible guidelines could include (based on @garrison_genomic_2019, @garrison_ge
...
 
-One [article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7895446/)  by @begay_weaving_2020 about the history of genetic studies of the Diné (or Navajo nation), another population that has faced genetic data misuse, state that the authors: 
+One [article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7895446/)  by @begay_weaving_2020 about the history of genetic studies of the Diné (or Navajo nation), another population that has faced genetic data misuse, state that the authors:
 
 > encourage researchers to consider cultural perspectives and traditional knowledge that has the potential to create stronger conclusions and better-informed, ethical, and respectful science.
 
 ## General Ethics Code
 
-maybe go into this: https://educationprojects.co.uk/medical-ethics-ethical-dilemmas-in-healthcare/
+Healthcare ethics, and by extension medical research ethics can be described in four pillars (this discussion from @melvin_medical_2020):
```
Comment (carriewright11) [inline 05-Data_Ethics.Rmd:172]: This is great, thanks!

File: 05-Data_Ethics.Rmd [hunk 2]
```diff
@@ -159,39 +159,76 @@ Possible guidelines could include (based on @garrison_genomic_2019, @garrison_ge
...
 
-and this amazing timeline:
-https://www.usuhs.edu/sites/default/files/2021-06/TimelineofLandmarkEventsinBioethics.pdf
+## Genetic Information Nondiscrimination Act of 2008 (GINA)
 
-- informed consent etc. [@commissioner_guide_2020]
+The Genetic Information Nondiscrimination Act prohibits employers and health insurance companies from using genetic information to discriminate against individuals.
+This data would be highly dangerous and valuable if in the wrong hands. Health insurance companies are prohibited from obtaining genetic test data and using it to inform their premiums or other healthcare coverage related decisions.
 
-Code of Medical Ethics (1957) - Legal Binding of Patient-Physician Confidentiality
```
Comment (carriewright11) [inline 05-Data_Ethics.Rmd:179]: is this somewhere else now as a note? sorry it is hard to see cause somehow the spacing is different so almost everything looks changed :P

File: 05-Data_Ethics.Rmd [hunk 3]
```diff
@@ -159,39 +159,76 @@ Possible guidelines could include (based on @garrison_genomic_2019, @garrison_ge
...
 
-2009 Health Information Technology for Economic and Clinical  Health (HITECH) Act
+This means as researchers or clinicians we need to keep these data secure and hidden. Data types protected under GINA include but are not limited to:
+
+- DNA sequencing or genotyping
+- RNA sequencing that shows genetic variation
+- Protein sequencing
+- Blood collection
+- Any type of biological material that would indicate heredity or any genetic variant
+
```
Comment (carriewright11) [inline 05-Data_Ethics.Rmd:213]: This is great! I actually planned to put this in the data privacy chapter - where we talk about genetic data.  what do you think about that?

Comment (cansavvy) [reply 05-Data_Ethics.Rmd:213]: Move this wherever you have envisioned it should be! No worries!

File: 05-Data_Ethics.Rmd (hunk 2 above)
Comment (cansavvy) [reply 05-Data_Ethics.Rmd:179]: Yeah I hate when it does that, I was about to start typing about this too which is why I deleted it, but I haven't yet added a description of this yet. I will add it back in for now

Comment (carriewright11) [reply 05-Data_Ethics.Rmd:179]: I had kinda hoped to have a general historical context/timeline section here.

File: 05-Data_Ethics.Rmd [hunk 4]
```diff
@@ -159,39 +159,76 @@ Possible guidelines could include (based on @garrison_genomic_2019, @garrison_ge
...
+Patients need to make their own decisions about their health. They should be equipped with all information about their health but then respected and supported in their decision.
+This is otherwise includes informed consent [@commissioner_guide_2020], which means before a patient can truly consent they need to be fully informed of the risks, ins and outs of any procedure, treatment, or research study participation.
+
+## Code of Medical Ethics (1957) - Legal Binding of Patient-Physician Confidentiality
+
+1996 Health Insurance Portability and Accountability Act (HIPAA) protected health information and identifiable health information must not be shared with anyone outside of certain covered entities without consent. Covered entities include: clinicians, insurance companies, and health care government agencies.
 
-and this amazing timeline:
-https://www.usuhs.edu/sites/default/files/2021-06/TimelineofLandmarkEventsinBioethics.pdf
```
[the remark this answers is not in the document]
Comment (cansavvy) [reply 05-Data_Ethics.Rmd:175]: I was working on building one but haven't added it in yet.

File: 05-Data_Ethics.Rmd (hunk 3 above)
Comment (carriewright11) [reply 05-Data_Ethics.Rmd:213]: OK cool! thanks

File: 05-Data_Ethics.Rmd (hunk 4 above)
[the remark this answers is not in the document]
Comment (carriewright11) [reply 05-Data_Ethics.Rmd:175]: Oh ok cool. I'd love to help with that

File: 05-Data_Ethics.Rmd (hunk 2 above)
Comment (carriewright11) [reply 05-Data_Ethics.Rmd:179]: I think there might be some other spots like that

Comment (cansavvy) [reply 05-Data_Ethics.Rmd:179]: Oh this header is moved up: line 193

File: 05-Data_Ethics.Rmd (hunk 4 above)
[the remark this answers is not in the document]
Comment (cansavvy) [reply 05-Data_Ethics.Rmd:175]: https://cloud.smartdraw.com/share.aspx/?pubDocShare=934B460036D53660C557F2FC651EA51CB15

**Comment by carriewright11** [comment]
I want to make the "timeline" section in the ethics chapter have a brief description of each of the things that I had listed/maybe what is now in the timeline. 

I moved GINA to the privacy chapter which is where I had intended it - I want to figure out what it's protection actually means for data sharing legally, I am still a bit confused about this. So I left it as a comment for now so I know where to modify it later.

**Comment by carriewright11** [comment]
merging this cause we have a bug in the URL checker for now

**Pull request merged**